test/functional/apps.js

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+
+let testAppPath, uiCatalogApp;
+
+// Had to make these two optional dependencies so the tests
+// can still run in linux
+if (process.platform === 'darwin') {
```

Comment:
`appium-support` has methods for this: https://github.com/appium/appium-support/blob/master/lib/system.js#L11